src/ngx_http_lua_socket_tcp.c

```diff
@@ -4908,65 +4901,30 @@ ngx_http_lua_socket_push_input_data(ngx_http_request_t *r,
     nbufs = 0;
     ll = NULL;
 
+    luaL_buffinit(L, &luabuf);
     for (cl = u->bufs_in; cl; cl = cl->next) {
+        ptrdiff_t chunk_size;
```

Comment:
@lneto Well, just point me to a place in the nginx core that uses `ptrdiff_t` for similar usage. Also point me to a place where the nginx core declares a local variable in an inner code block scope (inside a C function).